src/asammdf/blocks/conversion_utils.py

```diff
@@ -379,7 +378,7 @@ def from_dict(conversion: dict[str, Any]) -> v4b.ChannelConversion:
     return conversion
 
 
-def to_dict(conversion: ChannelConversionType) -> Union[dict, None]:
+def to_dict(conversion: ChannelConversionType) -> dict | None:
```

Comment:
Doesn't this require Python >= 3.10 ?

Reply:
Yes it does, but ruff ignores it, because the file contains `from __future__ import annotations`.

Reply:
Ok, let me know when it is ready for merging

Reply:
I'd say it's ready. This little change resulted in 73 modified files, that's enough for one PR 😄